src/netspresso_trainer/dataloaders/augmentation/custom/image_proc.py

```diff
@@ -622,17 +759,24 @@ def __init__(
         self,
         base_size: List,
         stride: int,
-        random_range: int,
+        random_range: Union[int, List],
         interpolation: str,
     ):
+        if isinstance(random_range, Sequence):
+            _check_sequence_input(random_range, 'random_range', req_sizes=(2,))
+            if random_range[0] > random_range[1]:
+                raise ValueError("Invalid side range provided {}.".format(random_range))
         self.base_size = base_size
         self.stride = stride
         self.random_range = random_range
         self.resize = Resize(size=base_size, interpolation=interpolation, max_size=None, resize_criteria=None)
         self.counter = 0
 
     def random_set(self):
-        delta = self.stride * random.randint(-self.random_range, self.random_range)
+        if isinstance(self.random_range, int):
+            delta = self.stride * random.randint(-self.random_range, self.random_range)
+        elif isinstance(self.random_range, Sequence):
+            delta = self.stride * random.randint(self.random_range[0], self.random_range[1])
```

Comment:
Sure! I made a mistake that random_range also can be a `Sequence`... It can be an integer when we adjust the base_size parameter (e.g., when we set the random_range [0,10], we can set the base size to be the scale of (0+10)/2 = 5 and make the random_range parameter to be an integer)! Thank you so much for your review👍